changelogs/unreleased/gh-6063-net-box-ping-error-throw.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/core
+
+* Fixed a bug when conn:ping() returned false for any exception 
+  been thrown. Now false is returned if there's no connection or 
```

Comment:
There are trailing whitespaces on first lines (can see them in `git diff/show`).